### ISSUE TITLE
Potential issue in scaling rewards in train_finetune.py.

### ISSUE THREAD

**Issue by ethanluoyc** [ikostrikov/implicit_q_learning#4]
Hi @ikostrikov,

thanks again for sharing this.

I have some questions on a potential issue in train_fientune when working with mujoco environments.

I noticed that rewards are not scaled for hopper, halfcheetah or walker2d during online fine-tuning. 
However, for these tasks, you normalized the rewards in the offline datasets. See e.g.,
https://github.com/ikostrikov/implicit_q_learning/blob/09d700248117881a75cb21f0adb95c6c8a694cb2/train_finetune.py#L82

Is this intended or I have misunderstood something. Many thanks!

**Comment by ikostrikov** [comment]
Hi @ethanluoyc,

Yes, this codebase doesn't support finetuning on these tasks. Please see the pytorch codebase: https://github.com/rail-berkeley/rlkit/tree/master/examples/iql

**Comment by ethanluoyc** [comment]
Thanks for the clarification. Closing this now.
